.cirrus.yml

```diff
@@ -508,6 +512,32 @@ container_integration_test_task:
     main_script: *main
     always: *int_logs_artifacts
 
+# Run the integration tests using the latest upstream build of netavark.
+netavark_integration_test_task:
+    name: "Netavark integration"  # using *std_name_fmt here is unreadable
+    alias: netavark_integration_test
+    only_if: *not_docs
+    skip: *branches_and_tags
+    depends_on:
+        - unit_test
+    gce_instance: *standardvm
+    env:
+        DISTRO_NV: ${FEDORA_NAME}
+        _BUILD_CACHE_HANDLE: ${FEDORA_NAME}-build-${CIRRUS_BUILD_ID}
+        VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+        CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+        TEST_FLAVOR: int
+        TEST_ENVIRON: host-netavark
+        NETAVARK_URL: "https://api.cirrus-ci.com/v1/artifact/github/containers/netavark/success/binary.zip?branch=${NETAVARK_BRANCH}"
+        NETAVARK_DEBUG: 0  # set non-zero to use the debug-mode binary
+        AARDVARK_URL: "https://api.cirrus-ci.com/v1/artifact/github/containers/aardvark-dns/build_and_test/binary.zip?branch=${AARDVARK_BRANCH}"
```

Comment:
***Note***: The `build_and_test` task will most likely be changing names when I convert aardvark CI to use VMs, necessitating a change here :confused:.  Not sure how best to deal with this yet.

Reply:
Ref: https://github.com/containers/aardvark-dns/pull/23

Reply:
Update: PR 23 will change the URL (task name -> "success" instead of "build_and_test").  So once that PR merges, I'll fix the URL here and re-push.